packages/angular/src/executors/module-federation-ssr-dev-server/lib/start-static-remotes.ts

```diff
@@ -0,0 +1,51 @@
+import { type ExecutorContext, workspaceRoot } from '@nx/devkit';
+import { type Schema } from '../schema';
+import fileServerExecutor from '@nx/web/src/executors/file-server/file-server.impl';
+import { join } from 'path';
+import { cpSync } from 'fs';
+import type { StaticRemotesConfig } from '@nx/webpack/src/utils/module-federation/parse-static-remotes-config';
+
+export function startStaticRemotes(
+  ssrStaticRemotesConfig: StaticRemotesConfig,
+  context: ExecutorContext,
+  options: Schema
+) {
+  if (ssrStaticRemotesConfig.remotes.length === 0) {
+    return;
+  }
+
+  // The directories are usually generated with /browser and /server suffixes so we need to copy them to a common directory
+  const commonOutputDirectory = join(workspaceRoot, 'tmp/static-remotes');
```

Comment:
Can we have add a listener for when original process exits / sigterms to remove this tmp directory?

This will need done for React too.

Reply:
This makes sense, we should do it.

Reply:
On second thought let's leave it, the process is being forcibly ended quicker than these have time to run. 
Let's take a look at enhancing them later on.